docs/index.bs

```diff
@@ -2875,6 +2875,7 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
 
               1. Wait for |task| to have executed or been discarded.
               1. Wait for the step labeled *WaitForAsynchronousExtensions* to complete.
+              1. If |installingWorker|'s [=list of router rules=] [=list/contains=] "fetch-event" and [=set of event types to handle=] does not [=set/contain=] <code>fetch</code>, then set |installFailed| to true.
```

Comment:
To make these link correctly:

```suggestion
              1. If |installingWorker|'s [=list of router rules=] [=list/contains=] "{{RouterSource/fetch-event}}" and [=set of event types to handle=] does not [=set/contain=] {{ServiceWorkerGlobalScope/fetch!!event}}, then set |installFailed| to true.
```

Reply:
Fixed. There remain some `<code>fetch</code>` originally in the spec, but I did not fix them in this PR.